egs2/swahili/asr1/run.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set="train_babel_alffa_gamayun_iwslt"
+valid_set="valid_alffa_iwslt"
+test_sets="test_iwslt_swa test_iwslt_swc test_iwslt_swa_raw test_iwslt_swc_raw"
+
+asr_config=conf/train_asr_conformer.yaml
+lm_config=conf/train_lm_transformer.yaml
+inference_config=conf/decode_asr.yaml
+
+./asr.sh \
+    --lang sw \
+    --ngpu 2 \
+    --nbpe 100 \
```

Comment:
Definite OK if your system is set as so. I'm just interesting if the bpe size is a bit small to the data (I remember the combination of all the data should be around 100 hours (correct me if I'm wrong)

Reply:
That's right, the data is 60 hours.

WER seems to increase when the BPE size is increased:

|nbpe|swa|swc|
|----|---|---|
|100| 27.7| 26.8|
|150| 26.8| 28.0|
|200| 28.6| 26.9|
|500| 28.6| 27.7|

Reply:
Interesting, thanks for your inforamtion!